packages/plugins/typescript/graphql-apollo/jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../../../jest.project')({	dirname: __dirname });
```

Comment:
Why do we need that?

Reply:
not sure, every typescript I checked had this.

Reply:
Ok let's keep it then.